data/mods/blazemod/items/vehicle/blaze_engine.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
No changes here beyond separating them out, I _think_ this is vanilla file structure?